Fix runtime of www/gitlab-ce

Allow omniauth 1.9.* to fix bundler check when gitlab starts.

--- omniauth-oauth2.gemspec.orig	2023-01-05 23:51:11 UTC
+++ omniauth-oauth2.gemspec
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.specification_version = 4
 
   s.add_runtime_dependency(%q<oauth2>.freeze, [">= 1.4", "< 3"])
-  s.add_runtime_dependency(%q<omniauth>.freeze, ["~> 2.0"])
+  s.add_runtime_dependency(%q<omniauth>.freeze, [">= 1.9", "< 3"])
   s.add_development_dependency(%q<bundler>.freeze, ["~> 2.0"])
 end
 
